lib/dynflow/executors/abstract/core.rb

```diff
@@ -35,6 +35,9 @@ def handle_event(event)
         end
 
         def work_finished(work)
+          if work.is_a?(Director::StepWorkItem) && ( wakeup_info = work.step.wakeup_info )
+            Dynflow.process_world.clock.ping ExecutionPlan::Steps::Suspended.new(work.step), wakeup_info[:time], wakeup_info[:event]
```

Comment:
Iirc `Dynflow.process_world` is not set for non-sidekiq deployments, however we can use `@world` directly